Dockerfile

```diff
@@ -1,79 +1,50 @@
-#
-# CLA Dockerfile
-#
-# https://github.com/dockerfile/nginx
-#
-# Pull base image.
-FROM phusion/baseimage:0.9.22
-
-LABEL name="Check If You Can Get Legal Aid (cla_public)" \
-      maintainer="LAA Get Access <laa-get-access@digital.justice.gov.uk>" \
-      version="1.0"
-
-ENV HOME /root
-
-# Install python and build packages
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
-    apt-get -y --force-yes install \
-      apt-utils \
-      build-essential \
-      g++ \
+FROM alpine:3.9
+
+RUN apk add --no-cache \
+      bash \
+      gettext \
+      nginx \
+      npm \
+      py2-pip \
+      supervisor \
+      tzdata \
+      uwsgi-python && \
+    adduser -D www-data -G www-data
+
+# To install python and nodejs dependencies
+RUN apk add --no-cache \
+      autoconf \
+      automake \
+      build-base \
       git \
       libffi-dev \
-      libpcre3 \
-      libpcre3-dev \
-      libpq-dev \
-      make \
-      nodejs \
-      python-dev \
-      python-pip \
-      python-software-properties \
-      software-properties-common \
-      tzdata
+      nasm \
+      openssl-dev \
+      python2-dev \
+      zlib-dev
 
-# Set timezone
-RUN ln -fs /usr/share/zoneinfo/Europe/London /etc/localtime
-
-# Install nginx
-RUN add-apt-repository ppa:nginx/stable && \
-    apt-get update && \
-    apt-get -y --force-yes install nginx-full && \
-    chown -R www-data:www-data /var/lib/nginx && \
```

Comment:
:no_entry: Alpine's nginx is using `/var/tmp/nginx` and it should be reflected in the pull request.